utt/cmd_report.py

```diff
@@ -56,54 +82,27 @@ def execute(args):
 ]
 
 
-def _activities_from_entries(entries_grouped_by_day):
-    activities_grouped_by_day = collections.defaultdict(list)
-
-    for date, entries in entries_grouped_by_day.items():
-        activities_grouped_by_day[date] = list(
-            _activities_from_entries_day(entries))
-
-    return activities_grouped_by_day
-
+def _collect_activities(start_date, end_date, entries):
+    start_datetime = datetime.datetime(start_date.year, start_date.month,
+                                       start_date.day)
+    end_datetime = datetime.datetime(end_date.year, end_date.month,
+                                     end_date.day, 23, 59, 59, 99999)
+    activities = []
+    for entry_pair in _pairwise(entries):
+        activity = Activity(entry_pair[0].datetime, entry_pair[1]).clip(
+            start_datetime, end_datetime)
+        if (activity.duration > datetime.timedelta() and
+                activity.name.name != HELLO):
```

Comment:
I would prefer if we don't rely on `"hello"`. The name of the first entry in a day can be anything. By convention it is `"hello"` since most users are probably using `utt hello`. However, there is a slight probability that explicitly checking for `"hello"` is a breaking change for a few users.

This doesn't need to be addressed, but if you find a way to avoid this breaking change that would be ideal.